cs/neon.texy

```diff
@@ -2,17 +2,17 @@ NEON
 ****
 
 .[perex]
-NEON is a human friendly data serialization language. [Try it on the sandbox |https://ne-on.org].
+NEON je snadno čitelný formát pro serializaci dat. [Vyzkoušejte jej v sandboxu |https://ne-on.org].
 
-It is similar to YAML. The main difference is that the NEON supports "entities" (so can be used e.g. to [DI configuration |configuring]) and tab characters for indentation. NEON syntax is a little simpler and the parsing is faster.
+Je podobný YAMLu. Hlavní rozdíl je v tom, že NEON podporuje "entity" (takže jej můžeme použít např. ke konfiguraci DI). K odsazování můžeme použít jak mezery, tak tabulátory. Syntaxe NEONu je v porovnání s YAMLem jednodušší a parsování je rychlejší.
 
 
-Encoding and decoding
-=====================
+Převod do NEONu a zpět
```

Comment:
@f3l1x Oosobně preferuji český překlad, i když se vytratí slovní vazba mezi nadpisem a popisovanými metodami. Česká dokumentace je primárně určena pro lidi, kteří anglické nerozumí, takže bych anglické výrazy omezil jen na případy, které nemají zažitý český ekvivalent.